Dockerfile

```
@@ -1,10 +1,10 @@
-FROM debian:buster
+FROM debian:bullseye
```

Maybe it'll work if it's unpinned?

```suggestion
FROM debian
```

---

Good thinking, works like a charm.